scss/os/_mixins.scss

```diff
@@ -61,7 +61,6 @@
     @content;
   }
 
-  .u-bg-#{$color}-text-a .u-link-style,
```

Comment:
this probably should be replaced with .btn.btn-link
I think the purpose is this colors the link to a color that doesnt clash with the background color (say if the background is blue, and the link is blue)